crates/cargo-contract/src/cmd/extrinsics/events.rs

```diff
@@ -103,16 +181,17 @@ pub fn display_events(
                     *field_ty,
                     event_data,
                 )?;
-                maybe_println!(
-                    verbosity,
-                    "{:width$}{}",
-                    "",
-                    format!("{}: {}", field_name.bright_white(), decoded_field),
-                    width = EVENT_FIELD_INDENT
-                );
+                let field = Field::new(field_name, decoded_field);
+                event_entry.fields.push(field);
             }
         }
+        events.push(event_entry);
     }
-    println!();
-    Ok(())
+
+    Ok(CallResult {
+        events,
+        verbosity,
+        output_type,
+        ..Default::default()
```

Comment:
Prefer to explicitly initialise all fields.

Reply:
I think using `Default::default()` allows in future override `Default` trait for types we want and specify the default value without changing manually editing multiple lines

Reply:
I disagree, if you add a new field which implements `Default` it will be easy to forget to initialize it properly. If you explicitly require each field to be initialized then the compiler will prevent this, if you just want the default value it's not much effort to add a `new_field: Default::default()`.